osu.Game/Screens/Edit/Editor.cs

```diff
@@ -84,30 +86,39 @@ public Editor()
                         new Container
                         {
                             RelativeSizeAxes = Axes.Both,
-                            Padding = new MarginPadding { Top = 5, Bottom = 5, Left = 10, Right = 10 },
-                            Child = new FillFlowContainer
+                            Padding = new MarginPadding { Vertical = 5, Horizontal = 10 },
+                            Children = new Drawable[]
                             {
-                                Name = "Bottom bar",
-                                RelativeSizeAxes = Axes.Both,
-                                Direction = FillDirection.Horizontal,
-                                Spacing = new Vector2(10, 0),
-                                Children = new[]
+                                timeInfo = new TimeInfoContainer
                                 {
-                                    timeline = new SummaryTimeline
-                                    {
-                                        Anchor = Anchor.Centre,
-                                        Origin = Anchor.Centre,
-                                        RelativeSizeAxes = Axes.Both,
-                                        Width = 0.65f
-                                    }
-                                }
+                                    Anchor = Anchor.CentreLeft,
+                                    Origin = Anchor.CentreLeft,
+                                    RelativeSizeAxes = Axes.Both,
+                                    Width = 0.17f
+                                },
+                                timeline = new SummaryTimeline
+                                {
+                                    Anchor = Anchor.Centre,
+                                    Origin = Anchor.Centre,
+                                    RelativeSizeAxes = Axes.Both,
+                                    Width = 0.65f
+                                },
+                                playback = new PlaybackContainer
+                                {
+                                    Anchor = Anchor.CentreRight,
+                                    Origin = Anchor.CentreRight,
+                                    RelativeSizeAxes = Axes.Both,
+                                    Width = 0.17f
+                                },

```

Comment:
It is my understanding that in this component, only the timeline should resize. I suggest using a `GridContainer` here where the first and last columns are absolute-sized.